# Resume

## Tech Skills

**Languages:** Java, Python, JavaScript, SQL (PostgreSQL), Bash, R  
**Libraries/Frameworks:** Spring Boot, Hibernate, Gradle, NodeJS, React, ExpressJS, D3.js  
**Concepts:** REST API, CI/CD, Infrastructure as Code (CloudFormation), ETL  
**Tools:** AWS, Git/Github, Jira

## Work Experience

**Team Lead, Global Alliance for Genomics and Health**  
Toronto, ON, Canada&emsp;&emsp;x 2021 - Present

* Foo

**Software Developer, Global Alliance for Genomics and Health**  
Toronto, ON, Canada&emsp;&emsp;April 2019 - x 2021

**Big Data Developer, Solsys Corporation**  
Toronto, ON, Canada&emsp;&emsp;March 2018 - April 2019

* Developed applications to detect and report real-time fraudulent transactions for bank client using Python and Splunk (big data analytics platform)
* Developed dashboards to monitor the health of bank applications deployed in the cloud
* Designed custom visualizations in D3.js to display server availability and response times
* Communicated with application teams and implemented requirements into new searches, reports, and visualizations

**Bioinformatician, Ontario Institute for Cancer Research**  
Toronto, ON, Canada&emsp;&emsp;September 2015 - March 2018

* Processed and analyzed large genomic datasets in a high-performance computing environment
* Developed tools in Python, R, and Bash to assess sample quality and determine samples that require resequencing
* Developed data visualization web applications using the Shiny package in R to share QC results with scientific collaborators
* Reviewed scientific literature to implement new bioinformatics software into our pipeline
* Presented research on new technologies with lab group to keep our pipeline updated and optimized

**Teaching Assistant, University of Waterloo**  
Waterloo, ON, Canada&emsp;&emsp;September 2013 - September 2015

* Taught molecular biology and genetics concepts to undergraduate students in tutorial
* Organized and supervised labs on nervous and cardiovascular system physiology
* Worked closely with students to ensure comprehensive understanding of course material
* Marked student lab reports, essays, and tests in a fair and unbiased manner
* Supervised and maintained control of midterm tests and exams

## Education

**Master of Science in Biology, University of Waterloo**  
Waterloo, ON, Canada&emsp;&emsp;September 2013 - September 2015

* Received NSERC Canada Graduate Scholarship (CGS-M) - $17 500
* Received University of Waterloo Presidentâ€™s Graduate Scholarship (PGS) - $10 000
* Designed and programmed a novel software analysis pipeline to detect likely instances of protein functional evolution using structural algorithms
* Taught Python coding seminars on bioinformatics analysis to students
* Co-authored paper on my thesis research

**Ontario Graduate Certificate in Bioinformatics, Seneca College**  
Toronto, ON, Canada&emsp;&emsp;September 2012 - June 2013

* Graduated with high honours (GPA over 3.95)
* Analyzed next-generation sequencing data to screen for genomic variants and somatic mutations using SAMtools
* Created dynamic web pages and scripts for querying biological databases, and designed graphical-user interfaces using Perl, SQL, Java, and R in the UNIX/Linux environment
* Designed and built a yeast strain database and querying tool for an academic lab

**Honours Bachelor of Science in Cell and Systems Biology, University of Toronto**  
Toronto, ON, Canada&emsp;&emsp;September 2008 - June 2012

* Graduated with high distinction (GPA over 3.50)
* Learned molecular biology, genetics, and biochemistry concepts, as well as how to interpret experimental data from primary research papers in these fields
* Cloned and expressed a gene of interest using molecular techniques
* Purified recombinant protein of interest by affinity chromatography
* Performed quantitative reverse transcription-PCR to create differential expression data
* Detected protein-protein interaction by yeast two-hybrid screen

## Publications

* Borozan I, Zaidi SH, Harrison TA, Phipps AI, ..., **Adams J**, ..., Peters U. Molecular and Pathology Features of Colorectal Tumors and Patient Outcomes Are Associated with Fusobacterium nucleatum and Its Subspecies animalis. Cancer Epidemiol Biomarkers Prev. 2022 Jan;31(1):210-220. doi: [10.1158/1055-9965.EPI-21-0463](https://doi.org/10.1158/1055-9965.EPI-21-0463). Epub 2021 Nov 4. PMID: [34737207](https://pubmed.ncbi.nlm.nih.gov/34737207/)

* Rehm HL, Page AJH, Smith L, **Adams JB**, ..., Birney E. GA4GH: International policies and standards for data sharing across genomic research and healthcare. Cell Genom. 2021 Nov 10;1(2):100029. doi: [10.1016/j.xgen.2021.100029](https://doi.org/10.1016/j.xgen.2021.100029). PMID: [35072136](https://pubmed.ncbi.nlm.nih.gov/35072136/)

* Yates AD, **Adams J**, Chaturvedi S, Davies RM, Laird M, Leinonen R, Nag R, Sheffield NC, Hofmann O, Keane TM. Refget: standardised access to reference sequences. Bioinformatics. 2021 Jul 14;38(1):299-300. doi: [10.1093/bioinformatics/btab524](https://doi.org/10.1093/bioinformatics/btab524). PMID: [34260694](https://pubmed.ncbi.nlm.nih.gov/34260694/)

* Zaidi SH, Harrison TA, Phipps AI, Steinfelder R, ..., **Adams JB**, ..., Peters U. Landscape of somatic single nucleotide variants and indels in colorectal cancer and impact on survival. Nat Commun. 2020 Jul 20;11(1):3644. doi: [10.1038/s41467-020-17386-z](https://doi.org/10.1038/s41467-020-17386-z). PMID: [32686686](https://pubmed.ncbi.nlm.nih.gov/32686686/)

* **Adams J**, Mansfield MJ, Richard DJ, Doxey AC. Lineage-specific mutational clustering in protein structures predicts evolutionary shifts in function. Bioinformatics. 2017 May 1;33(9):1338-1345. doi: [10.1093/bioinformatics/btw815](https://doi.org/10.1093/bioinformatics/btw815). PMID: [28052926](https://pubmed.ncbi.nlm.nih.gov/28052926/)

* Mansfield MJ, **Adams JB**, Doxey AC. Botulinum neurotoxin homologs in non-Clostridium species. FEBS Lett. 2015 Jan 30;589(3):342-8. doi: [10.1016/j.febslet.2014.12.018](https://doi.org/10.1016/j.febslet.2014.12.018). Epub 2014 Dec 23. PMID: [25541486](https://pubmed.ncbi.nlm.nih.gov/25541486/)

## Other Professional Development

**Certificate in Big Data Analytics, York University**  
Toronto, ON, Canada&emsp;&emsp;January 2019 - March 2019

* Solved analytical problems with machine learning and related them to business goals
* Developed and tested machine learning models in Python and R, including regression, decision tree, random forest, k-means clustering, and hierarchical clustering

**Informatics on High-throughput Sequencing Data, Canadian Bioinformatics Workshops**  
Toronto, ON, Canada&emsp;&emsp;June 2015

* Aligned raw read data to reference genome using Amazon Cloud Computing resources
* Detected small variants and structural variants using command-line tools
* Visualized genomic variants using the Integrated Genomic Viewer
* Designed custom sequencing analysis workflows on a graphical interface using the Galaxy web server

**Informatics for RNA-seq Analysis, Canadian Bioinformatics Workshops**  
Toronto, ON, Canada&emsp;&emsp;June 2015

* Assessed quality of Illumina raw read data using FastQC
* Performed transcriptome analysis using the Tuxedo suite of tools
* Quantified mature transcript levels and performed differential expression analysis between tissues using Cufflinks
* Detected alternative splicing patterns and discovered potential isoforms using Cufflinks
